src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/BaseDataDonationViewModel.swift

```diff
@@ -0,0 +1,104 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+import OpenCombine
+
+/*** this protocol gets used to provide different view model classes for the dataDonationViewController */
+
+protocol DataDonationViewModelProtocol {
+
+	/// use these 3 to definde @published in a prorocol
+	var reloadTableView: Bool { get }
+	var reloadTableViewPublished: OpenCombine.Published<Bool> { get }
```

Comment:
Is this used somewhere?

Reply:
I removed the published property it's not need.